setup_dev.sh

```diff
@@ -31,7 +31,9 @@ sed -i "s@^  :password.*@  :password => '$mysql_congress_forms',@" config/databa
 
 HOME=/home/vagrant sudo -u vagrant /bin/bash <<EOF
 echo "Setting up RVM and Ruby..."
-curl -sSL https://get.rvm.io | bash -s stable
+gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+curl -sSL https://rvm.io/mpapis.asc | gpg --import -
```

Comment:
I don't think this line is needed if line 34 is included.

The question is, is line 34 or this one more secure?  Line 34 includes the whole fingerprint, but no transport-layer encryption.  If gpg asks for a key with the entire fingerprint, and then the server responds with an entirely different key, does gpg just blindly accept it?  Or does it try to verify that this key matches, and since it doesn't, reject it?  If the latter, that's very bad - a MITM could easily perform an attack and deliver a false key.  That's the case, this line is preferred and *not* 34.

We could get the best of both worlds, and in the $DEPENDENCIES add `gnupg2`, and then change the above line to:

`gpg2 --keyserver hkps://hkps.pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3`

and then just remove this line.